packages/flutter/lib/src/material/theme.dart

```diff
@@ -34,14 +36,22 @@ class Theme extends InheritedWidget {
   /// Specifies the color and typography values for descendant widgets.
   final ThemeData data;
 
+  /// True if this theme shadows the one created by the [MaterialApp].
```

Comment:
you should add some color here saying why you would set this (or rather, that you wouldn't), how it's used, etc.